Sources/Operators/FromAsync.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  FromAsync.swift
+//  CombineExt
+//
+//  Created by Thibault Wittemberg on 2021-06-15.
+//  Copyright © 2021 Combine Community. All rights reserved.
+//
+
+#if canImport(Combine)
+import Combine
+
+@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
```

Comment:
I believe this can be: 
```swift
#if compiler(>=5.5) && canImport(_Concurrency)
```
rather than forcing iOS15

https://www.swiftbysundell.com/special/swift-concurrency-backward-compatibility/